app/javascript/Components/Result/left_pane.jsx

```diff
@@ -141,30 +145,27 @@ export class LeftPane extends React.Component {
             )}
 
             <TestRunTable
-              result_id={this.props.result_id}
-              submission_id={this.props.submission_id}
-              assignment_id={this.props.assignment_id}
-              grouping_id={this.props.grouping_id}
+              result_id={this.context.result_id}
+              course_id={this.context.course_id}
+              assignment_id={this.context.assignment_id}
+              grouping_id={this.context.grouping_id}
+              submission_id={this.context.submission_id}
               instructor_run={this.props.instructor_run}
               instructor_view={!this.props.student_view}
-              course_id={this.props.course_id}
             />
           </div>
         </TabPanel>
-        <TabPanel forceRender={!LeftPane.disableFeedbackFilesPanel(this.props)}>
+        <TabPanel forceRender={!this.disableFeedbackFilesPanel()}>
           <FeedbackFilePanel
-            assignment_id={this.props.assignment_id}
             feedbackFiles={this.props.feedback_files}
-            submission_id={this.props.submission_id}
-            course_id={this.props.course_id}
             loading={this.props.loading}
           />
         </TabPanel>
-        <TabPanel forceRender={!LeftPane.disableRemarkPanel(this.props)}>
+        <TabPanel forceRender={!this.disableRemarkPanel()}>
           <div id="remark_request_tab">
             <RemarkPanel
-              result_id={this.props.result_id}
-              assignment_id={this.props.assignment_id}
+              result_id={this.context.result_id}
```

Comment:
Same comment as above, `RemarkPanel` is only used under `Result`, so you shouldn't need to pass in these props any more